common/static/common/js/discussion/utils.js

```diff
@@ -386,8 +386,7 @@
                         /*
                          bug fix, ordering is off
```

Comment:
@noraiz-anwar  and @awaisdar001  , trust your judgement here, just be sure to update that comment